README.md

```diff
@@ -19,97 +20,167 @@ This WordPress project is intended to provide:
+# These are for local development only.
+```
 
-- Create the MySQL user and database using the MariaDB monitor.
+- **Install Automated Test Suite**
 
-  ```
-  mysql -uroot
-  CREATE DATABASE chappress_dev;
-  GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
-  FLUSH PRIVILEGES;
-  EXIT
-  ```
+Navigate to project folder root: `/chap-press`
 
-- Download WordPress and run the server.
+```
+composer install
+#installs depedency from composer.json
 
-  ```
-  git clone git@github.com:chapmanu/chap-press.git
-  cd chap-press/public
-  cp -v wp-config.php{-dist,}
+echo "alias codecept=./vendor/bin/codecept" >> ~/.bash_profile  
+#creates an alias
 
-  # The wp-config file already contains the database information to get started.
-  # These are for local development only.
+source ~/.bash_profile 
 
-  php -S localhost:8000
-  ```
+codecept --version  
+```
 
-  Server should be running in the terminal displaying PHP version, port location, and document root.
+Codeception executed as `codecept` or `./vendor/bin/codecept`
 
-  Go to your browser and enter the url port:
-  ```
-  http://localhost:8000
-  ```
+- **Run the Server**
 
-  Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+Navigate to the `/public` directory and run `php -S localhost:8000`
+
+
+Go to your browser and enter the url port: `http://localhost:8000`
+
+Server should be running in the terminal displaying PHP version, port location, and document root.
+Finish the final instructions through the WordPress installation. The WordPress admin panel should be displayed and the user should have full access to developing in a local environment.
+
+***
 
 ## Debugging
 
-  WordPress comes with specific debug systems designed to simplify the process.
-  The following are meant only for local testing and staging installs.
+WordPress comes with specific debug systems designed to simplify the process.
+The following are meant only for local testing and staging installs.
+
+The following code is already set in the `wp-config.php` file.
+It will log all errors, notices, and warnings to a file called `debug.log` in the wp-content directory.
+It will also hide the errors so they do not interrupt page generation.
+
+<br/>
+
+| Command | Description |
+| --- | --- |
+| `define( 'WP_DEBUG', true );` | Enable WP_DEBUG mode |
+| `define( 'WP_DEBUG_LOG', true );` | Enable Debug logging to `/wp-content/debug.log` |
+| `define( 'WP_DEBUG_DISPLAY', false );` | Disable display of errors and warnings |
+| `@ini_set( 'display_errors', 0 );` | // |
+| `define( 'SCRIPT_DEBUG', true );` | Uses unminified versions of core JS and CSS files |
+
+***
+
+## Automated Testing
+
+### Run Automated Tests
+
+1. **Run Local Server** at `/public` directory: `php -S localhost:8000`
```

Comment:
Yes, this isn't acceptable. Unless it's absolutely necessary, in which case I'll create an issue and we can come back to it.

One of our Rails apps requires running `foreman start` before running tests. It's a minor nuisance but if it can be avoided it should be avoided. This is one of those [mise en place](https://en.wikipedia.org/wiki/Mise_en_place) things I'm insistent about.

Timebox the remainder of today to address this. If you can't crack it, open an issue and we'll move forward.